src/pilgrim/ui/screens/diary_list_screen.py

```
@@ -206,29 +206,17 @@ def action_new_diary(self):
         """Action to create new diary"""
         self.app.push_screen(NewDiaryModal(),self._on_new_diary_submitted)
 
-    def _on_new_diary_submitted(self,result):
-        self.notify(str(result))
-        if result:
-            self.notify(f"Creating Diary:{result}'...")
-            self.call_later(self._async_create_diary,result)
+    def _on_new_diary_submitted(self, result):
+        """Callback after diary creation"""
+        if result:  # Se result não é string vazia, o diário foi criado
+            self.notify(f"Returning to diary list...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)